index.js

```diff
@@ -2,14 +2,65 @@ const Vis = require("./graphvis");
 const { Module, render } = require("./visRenderer");
 
 let Sequelize;
-const relationship = ({ source, target, associationType, as }) => {
-  const typeString = {
-    BelongsTo: `[arrowtail=crow, arrowhead=none, dir=both, arrowsize=0.60]`,
-    BelongsToMany: `[arrowtail=crow, arrowhead=crow, dir=both, arrowsize=0.60]`
-  }[associationType];
-  if (typeString) {
-    return `"${source.name}" -> "${target.name}" ${typeString}`;
-  }
+
+const relationships = (associations, customArrowShapes = {}, arrowSize = 0.6) => {
+  const arrowShapes = Object.assign({
+    BelongsToMany: ['none', 'crow'],
+    BelongsTo: ['crow', 'none'],
+    HasMany: ['none', 'crow'],
+    HasOne: ['none', 'none'],
+  }, customArrowShapes);
+
+  const groupedAssociations = associations.reduce((result, association) => {
+    if (typeof result[association.associationType] === 'undefined')
+      result[association.associationType] = [association];
+    else
+      result[association.associationType] = [...result[association.associationType], association];
+
+    return result;
+  }, {});
+
+  const mappings = [  // Follow this heirarchy to ensure the arrows are set up correctly, eg. HasOne specifications override BelongsTo
```

Comment:
What does mappings really mean?

Reply:
Lol good question, but I'm not sure what variable name would work better. I'm just using @wwmoraes' naming here since I forked from him.

Reply:
I came up with some new variable names for this whole section in the next commit that are hopefully a little bit more descriptive.